### DISCUSSION TITLE
Will the Trials 1.20.1 branch receive fabric support on a later date?

### DISCUSSION THREAD

**Discussion by Domon3rt** [IzzelAliz/Arclight#1297] [Q&A]
Hello, I would first like to extend my gratitude as I think Arclight is beyond amazing and is a testament to what some developers can achieve. I do not develop myself but I couldn't imagine some of the difficulties and tribulations involved. So I genuinely thank you and on behalf of the rest of us for your time and service to the community.

Onto my simple question. Will 1.20.1 Trials receive fabric support?

I was planning on attempting using FFRAPI and Sinytra Connector together to bridge the gap for Fabric/Forge on 1.20.1 Trials. This did not work unfortunately and I am not versed in coding whatsoever. I'm more than sure you've already are familiar with them but if it provides any sort of convenience;

https://www.curseforge.com/minecraft/mc-mods/forgified-fabric-api
https://www.curseforge.com/minecraft/mc-mods/sinytra-connector
https://www.curseforge.com/minecraft/mc-mods/connector-extras

I only ask because 1.20.1 in terms of mods are at a particular milestone for the modding community alongside 1.20.4, which is catching up or offering it's own unique mods.

Thank you for your time. Keep being awesome IzzelAliz.

**Comment by TheoClouds** [comment]
Also would really like this!
